bedrock/careers/templates/careers/benefits.html

```diff
@@ -137,9 +137,9 @@ <h2 class="c-careers-section-title">
             Mozilla is a private company, so our compensation isn’t tied to stock
             options or equity plans. Instead, we offer generous, performance-based
             bonus plans to all regular employees to underscore that we share in
-            our success as one team. As for retirement savings for U.S. and
-            Canadian employees, Mozilla contributes a percentage of your eligible
-            base salary each year to the 401(k) Plan/RRSP, with 100% vesting.
+            our success as one team. As for retirement savings for US and Canadian employees,
+            Mozilla contributes a % of your eligible base salary each year to the 401(k) Plan/RRSP
```

Comment:
I think MJ would be fine if we kept percentage rather than use the shorthand

```suggestion
            Mozilla contributes a percentage of your eligible base salary each year to the 401(k) Plan/RRSP
```